Add keys module reference - Closes #867 (#880)
LiskHQ/lisk-docs: modules/ROOT/pages/keys-module.adoc

= The keys module

> The Keys module handles all logic related to the signatures.

Reviewer: ```suggestion
The Keys module handles all logics related to the signatures.
```

> It should verify the signatures based on the multi-signature rules including non-multi-signature accounts.
> It also handles the registration of multi-signature accounts.

Reviewer: ```suggestion
It is responsible for verifying signatures of a transaction from multi-signature and non-multi-signature accounts.
It also handles the registration of multi-signature accounts.
```
